staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -315,15 +316,91 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object) (runti
 			return nil, err
 		}
 	}
+	if includeUninitialized {
```

Comment:
My concern would be that watch semantics on a future store demand something that the API server can't correctly support.  I don't want the rest layer having to know about watch details, what the best mechanism is to wait, etc.